docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,306 @@
+
+* id: A unique identifier of the course; a serialized representation of the
+  opaque key identifying the course.
+
+* media: An object that contains named media items, including
+
+  * course_image: An image to show for the course
+
+    * uri: The location of the image
+
+  * course_video: A video about the course
+
+    * uri: The location of the video
+
+  * image: URLs for images in the course, including "raw", "small" or "large"
+    versions.
+
+* name: The name of the course.
+
+* number: The catalog number of the course.
+
+* org: The name of the organization that owns the course.
+
+* overview: An HTML textual description of the course.
+
+* short_description: A textual description of the course.
+
+* start: The date on which the course begins.
+
+* start_display: The readably formatted start date of the course.
+
+* start_type: Indicates how `start_display` is set. Possible values are:
+
+  * "string": manually set
+  * "timestamp": generated form `start` timestamp
+  * "empty": the start date should not be shown
+
+* pacing: The type of course pacing for this course. Possible values:
+  instructor, self
+
+* course_id: Course key. This field might be returned but is deprecated. You
+  should use ``id`` instead.
+
+* course_id: Course key. This field might be returned but is deprecated. You
+  should use ``id`` instead.
+
+================
+Return Values
+================
+
+* 200 on success with above fields.
+
+* 400 if an invalid parameter was sent, or if the username was not provided
+  for an authenticated request.
+
+* 403 if a user who does not have permission to masquerade as another user
+  specifies a username other than their own.
+
+* 404 if the course is not available or cannot be seen.
+
+==============================================================
+Example Response Showing a List of Courses Visible to a User
+==============================================================
+
+.. code-block:: json
+
+    {
+
+     "blocks_url":"/api/courses/v1/blocks/?course_id=edX%2Fexample%2F2012_Fall",
+     "media": {
+        "course_image": {
+        "uri": "/c4x/edX/example/asset/just_a_test.jpg",
+        "name": "Course Image"
+                        }
+              },
+      "description": "An example course.",
+      "end": "2015-09-19T18:00:00Z",
+      "enrollment_end": "2015-07-15T00:00:00Z",
+      "enrollment_start": "2015-06-15T00:00:00Z",
+      "course_id": "edX/example/2012_Fall",
+      "name": "Example Course",
+      "number": "example",
+      "org": "edX",
+      "start": "2015-07-17T12:00:00Z",
+      "start_display": "July 17, 2015",
+      "start_type": "timestamp"
+    }
+
+
+.. _Get the details for a specified course:
+
+*****************************************
+Get Details for a Specified Course
+*****************************************
+
+The endpoint to get the details for a specified course is
+``/api/courses/v1/courses/{course_key}/``.
+
+=====================
+Use Case
```

Comment:
I'd omit this section. It doesn't add to the description at the top level of the section.